test/plugin/base_test.rb

```diff
@@ -1065,14 +1126,14 @@ def setup_ec2_metadata_stubs
 
   def setup_auth_stubs
     # Used when loading credentials from a JSON file.
-    stub_request(:post, 'https://www.googleapis.com/oauth2/v3/token')
+    stub_request(:post, 'https://www.googleapis.com/oauth2/v4/token')
```

Comment:
Yes, it has to do with an upgrade of the `google-gax` gem that was done by the `google-cloud-logging` gem on 2/18/17, see: https://github.com/GoogleCloudPlatform/google-cloud-ruby/commit/4e4c7f42e43533d729aa65dd5a8fde47160a8c02

That `google-gax` gem version depends on `googleauth` "~> 0.5", see https://rubygems.org/gems/google-gax/versions/0.8.0

And the `googleauth` gem 0.5.2 on 7/19/17 (so pretty recently) did an upgrade to use the /v4/ auth endpoint, see https://github.com/google/google-auth-library-ruby/blob/master/CHANGELOG.md#052-20170719

What I did was pin the `googleauth` gem to `0.5.1`, and that did allow me to switch this back to `/v3/` while still upgrading the `google-cloud-logging` gem so it gets the new `trace` field.